### PR TITLE
LIME-536: Fix extra achievements

### LINKED ISSUE #536: BUG: Wrong extra achievements are received by user
### Describe your issue

Wrong extra achievements are received by user

### Steps to reproduce

1. Login under same account in Postman and in browser in https://live-in-motion-everyday.com/
2. Add workout in Postman with activityType "cycling" and distance "5000"
3. Pay attention to achievements added.
4. Add workout in Postman with activityType "running", distance "42000" and  dates in workoutStartedAt and workoutEndedAt that are not weekends
5. Pay attention to achievements added.
6. Add workout in Postman with activityType "walking",  distance "10000" and values of workoutStartedAt and workoutEndedAt that result in 5 h duration
7. Pay attention to achievements added.

### What was the actual result?

All achivements are received according to workouts

### What was the expected result?

Wrong extra achievements are received by user
for cycling - 1000 m height
for running - 10 km per weekend
for walking - 5 h per weekend

### Put here any screenshots or videos (optional)

_No response_

### What OS are you seeing the problem on?

Windows

### What browsers are you seeing the problem on?

Chrome

### Severity

Critical (S2)

### PR DESCRIPTION
**Pull request by SergiiTernytskyi**

### CODE REVIEW

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
why we change it? why we dont have half marathon?

**Comment by dimapopovych** [comment]
@nikita-remeslov 
we are not storing time restrictions in database at all. These are only in achievements names. Currently, there is ability to calculate only achievements for total [STAT] per whole time. Need to change database structure/seeds/calculation logic to handle it.

**Comment by SergiiTernytskyi** [comment]
> why we change it? why we dont have half marathon?

We have half marathon. There just was 21 km - half marathon and 42 km- half marathon.

**Comment by dimapopovych** [comment]
@SergiiTernytskyi 
Could you please remove all achievements with time restrictions and replace them with total [stat] achievements? This change is necessary to avoid confusion, as there are still a couple with incorrect names based on how they will be calculated.
```
{
        name: 'First 1 hour run',
        activityType: ActivityType.RUNNING,
        requirement: 60,
        requirementMetric: Metric.MINUTES,
    },
    {
        name: '100 km in a month',
        activityType: ActivityType.RUNNING,
        requirement: 100,
        requirementMetric: Metric.KILOMETERS,
    },
    {
        name: '200 km in a month',
        activityType: ActivityType.RUNNING,
        requirement: 200,
        requirementMetric: Metric.KILOMETERS,
    },
```

Like these, for example

File: backend/src/seeds/achievements.ts [hunk 1]
```diff
@@ -10,153 +10,165 @@ const Achievements = [
...
     {
-        name: 'First 21 km (half marathon)',
+        name: 'Running Milestone: 21 km (half marathon)',
         activityType: ActivityType.RUNNING,
         requirement: 21,
         requirementMetric: Metric.KILOMETERS,
     },
     {
-        name: 'First 42 km (half marathon)',
+        name: 'Running Milestone: 42 km (marathon)',
```
Comment (dimapopovych) [inline backend/src/seeds/achievements.ts:79]: remove half-marathon/marathon, and replace with just 50/100/200 km

File: backend/src/seeds/achievements.ts [hunk 2]
```diff
@@ -10,153 +10,165 @@ const Achievements = [
...
+    },
+    {
+        name: 'Walking Milestone: 100 km',
+        activityType: ActivityType.WALKING,
+        requirement: 100,
         requirementMetric: Metric.KILOMETERS,
     },
     {
-        name: '500 kcal per walk',
+        name: 'Calories Milestone: 1000 kcal burned',
```
Comment (dimapopovych) [inline backend/src/seeds/achievements.ts]: Walking Milestone: 100 kcal burned

**Review by dimapopovych — DISMISSED** [review]

**Review by anton-otroshchenko — APPROVED** [review]

**Review by dimapopovych — APPROVED** [review]

**Pull request merged**